python/tvm/driver/tvmc/runner.py

```diff
@@ -530,58 +531,82 @@ def run_module(
             assert device == "cpu"
             dev = session.cpu()
 
-        # TODO(gromero): Adjust for micro targets.
-        if profile:
-            logger.debug("Creating runtime with profiling enabled.")
-            module = debug_executor.create(tvmc_package.graph, lib, dev, dump_root="./prof")
+        if tvmc_package.use_vm:
```

Comment:
@jwfromm is there a way to get the inputs via the VM like in the graph executor path so we don't need to rely on external inputs being passed in here?

Reply:
sounds good -- i'll keep the assertion below then!